sqlglot/parser.py

```diff
@@ -7049,6 +7051,34 @@ def _parse_alter(self) -> exp.Alter | exp.Command:
 
         return self._parse_as_command(start)
 
+    def _parse_analyze(self) -> exp.Analyze | exp.Command:
+        start = self._prev
+        kind = None
+        this: exp.Expression | None = None
+        partition = None
+
+        if self._match(TokenType.TABLE):
+            kind = TokenType.TABLE.name
```

Comment:
Why are we accessing `kind` like this? Should we instead hardcode `"TABLE"` like in L7065?